src/src/pages/capabilities/myCapabilities.module.css

```diff
@@ -5,7 +5,6 @@
 }
 
 .costs{
-  width: 100%;
```

Comment:
We are slowly getting away from the world where table content is responsible for knowing how the table works. This is nice!